pebblo/app/models/models.py

```diff
@@ -172,3 +172,19 @@ class LoaderDocs(BaseModel):
 class LoaderDocResponseModel(BaseModel):
     docs: List[LoaderDocs] = []
     message: Optional[str] = None
+
+
+class DiscoverAIApps(BaseModel):
+    name: str
+    description: Optional[str]
+    owner: str
+    instanceDetails: Optional[InstanceDetails]
+    framework: Optional[FrameworkInfo]
+    lastUsed: datetime
+    pebbloServerVersion: Optional[str]
+    pebbloClientVersion: Optional[str]
+
+
+class DiscoverAIAppsResponseModel(BaseModel):
+    ai_apps_data: DiscoverAIApps | None
```

Comment:
Python version 3.9 does not allow writing union types as X | Y. You can use Optional here.

Reply:
sure